spec/spec_helper.rb

```diff
@@ -11,35 +12,29 @@
 require 'napa'
 require 'napa/rspec_extensions/response_helpers'
 
+module NapaSpecClassHelpers
+  # Apparently Thor's last name is Odinson... who knew?!
+  # http://marvel.com/universe/Thor_(Thor_Odinson)
+  # http://www.infinitelooper.com/?v=4D7cPH7DHgA#/170;230
+  def tell_me_mr_odinson_what_good_is_a_phone_call_if_youre_unable_to_speak?
```

Comment:
Maybeh. But since the function is reveling in melting shut Thor's loud smug mouth, I think it fits. I included a video for context.